### ISSUE TITLE
“Close” graphical buttons with insufficient accessible names [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#260]
The “close” graphical buttons found on the “Select your country”, “Gender identity”, “Age range”, “Race”, “Ethnicity” overlays accessed from the My Health Log About screen all have the same insufficient accessible name of “Close choose…”, which does not accurately describe the purpose of the button.

![“Close” graphical button focused by VoiceOver and announced as “Close Choose…”](https://user-images.githubusercontent.com/4437084/92043904-0dea2400-ed4b-11ea-8071-1b7d10bcb7de.png)
 
The “close” graphical button found on the “Select your country” overlay accessed from the App Settings -> My Health Log screen has the insufficient accessible name of “Close placeholder”.

![“Close” graphical button focused by VoiceOver and announced as “Close placeholder”](https://user-images.githubusercontent.com/4437084/92043910-104c7e00-ed4b-11ea-8a92-4b9dad20f097.png)

Ensure all the “close” buttons have an accessible name that adequately describes the purpose of the button (.e.g, “Close choose your gender identity”)

**Comment by floridemai** [comment]
@willslone This has now been addressed
